README.md

```diff
@@ -1,5 +1,5 @@
 # Eclipse Che documentation project
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/f?url=https://github.com/eclipse/che-docs)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/#https://github.com/eclipse/che-docs)
```

Comment:
Will the migration from `che.osio` to `workspaces.openshift.com` (based on CRW) break this?

Reply:
yes, I had an idea that we must migrate to workspaces, but CRW uses the old dashboard.
Once workspace.openshift.com use CRW 2.8 (where we plan use a new dashboard) we should be fine to migrate there.

Or we can migrate on workspaces with the old format, while documentation will contain description of the new, since it's upstream documentation and it's available starting from 7.26.

Do you think we should revert changes in README.md?

Reply:
@sleshchenko in 10 days every request to che.openshift.io will be redirected to workspaces.openshift.com that will be running CRW 2.5 or 2.6.

Reply:
I think it's not only one place where we haven't migrated yet, despite of many Illya's welcoming.
I'll migrate Che Docs to workspaces on monday, after Huge PR related to dashboard next is merged.

Reply:
Ok thank @sleshchenko for your help. Just a clarification:
- replacing `che.openshift.io` with `workspaces.openshift.com` is not required (we will have redirection) but URLs with `#` won't work on `workspaces.openshift.com`
- @ibuziuk is supposed to update the documentation next week, hence he may take care of these fixes on che-docs